tests/strategy/test_strategy_loading.py

```diff
@@ -218,6 +218,7 @@ def test_strategy_override_process_only_new_candles(caplog, default_conf):
 def test_strategy_override_order_types(caplog, default_conf):
     caplog.set_level(logging.INFO)
 
+    # TODO-lev: Maybe change
```

Comment:
Leverage is going to be a callback, not a configuration setting (`enable-leverage` might be - but i doubt we need that here as the logic for config/strategy overrides is well tested and identical for all settings). 

Having leverage a static (config) setting would not work - simply because "elsewhere" you're showing that allowed leverage will differ between pairs and traded volume.